src/test/java/BoardTest.java

```diff
@@ -79,4 +79,23 @@ public void testKnowsWhatCellsAreStillAvailable() {
         List availableMoves = board.availableMoves();
         assertFalse(availableMoves.isEmpty());
     }
+
+    @Test
+    public void testReturnsTheMarkAtARequestedPosition() {
+        Board board = new Board();
```

Comment:
How about removing some of the boilerplate setup that all tests are currently duplicating?

Reply:
✅ used `@Before` PR #6